pkg/controller/replication_controller.go

```diff
@@ -64,6 +64,10 @@ const (
 	// 3000 pods. Just creation is limited to 30qps, and watching happens with
 	// ~10-30s latency/pod at scale.
 	ExpectationsTimeout = 6 * time.Minute
```

Comment:
Yeah, why not fix it now, since the TODO will be truthful-but-misleading after this merges.